beesdoo_portal_shift/data/ir.model.access.csv

```diff
@@ -0,0 +1,6 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_beesdoo_shift_template,Public Read Template,beesdoo_shift.model_beesdoo_shift_template,,1,0,0,0
+access_beesdoo_shift_shift,Public Read Shift,beesdoo_shift.model_beesdoo_shift_shift,,1,0,0,0
+access_beesdoo_shift_type,Public Read Type Shift,beesdoo_shift.model_beesdoo_shift_type,,1,0,0,0
+access_beesdoo_shift_planning,Public Read Planning,beesdoo_shift.model_beesdoo_shift_planning,,1,0,0,0
+access_beesdoo_cooperative_status_history,Public Read Cooperative Status History,beesdoo_shift.model_cooperative_status_history,,1,0,0,0
```

Comment:
Dont give read access to public user. Instead use sudo() to instanciate function as admin